docs/resources/user.md

```diff
@@ -27,15 +27,18 @@ resource "wallarm_user" "user" {
 
 ## Argument Reference
 
-* `email` - (Required) User email. The value will be used as the username for authentication in Wallarm Console.
-* `realname` - (Required) The first and last name of the user.
-* `permissions` - (Required) User role. Can be one of: `admin`, `analyst`, `deploy`, `read_only`, `global_admin`, `global_analyst`, `global_read_only`. Roles description is available in the [Wallarm official documentation](https://docs.wallarm.com/user-guides/settings/users/#user-roles).
-* `password` - (Optional) User password. If the value is not specified, it will be generated automatically and returned in the attribute `generated_password`.
-* `phone` - (Optional) User phone number.
-* `client_id` - (Optional) ID of the client to apply the rules to. The value is required for multi-tenant scenarios.
+* `email` - (**required**) user email. The value will be used as the username for authentication in Wallarm Console.
+* `realname` - (**required**) the first and last name of the user.
+* `permissions` - (**required**) user role. Can be one of: `admin`, `analyst`, `deploy`, `read_only`, `global_admin`, `global_analyst`, `global_read_only`. Roles description is available in the [Wallarm official documentation](https://docs.wallarm.com/user-guides/settings/users/#user-roles).
```

Comment:
check the current list of roles and make sure this list is actual

Reply:
These are the roles supported by the provider. If any of them don't work anymore, please let me know. We're going to need to fix the code.